src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -156,4 +256,16 @@ private function isCommentLastLineToken(Tokens $tokens, $index)
 
         return $content !== ltrim($content, "\r\n");
     }
+
+    /**
+     * Checks if token is new line.
+     *
+     * @param Token $token
+     *
+     * @return bool
+     */
+    private function isNewLine(Token $token)
+    {
+        return trim($token->getContent(), " \r\t\0\x0B") === "\n";
```

Comment:
That's right. And that's not what `isNewLine` does, right? So... this is resolved?